src/store/zap/actions.js

```diff
@@ -709,33 +712,35 @@ export async function updateSelectedEndpointType(
           setEventStateLists(context, res.data || [])
         })
     )
-
     p.push(
-      axiosRequests
-        .$serverGet(
-          `${restApi.uri.deviceTypeClusters}${endpointTypeDeviceTypeRefPair.deviceTypeRef}`
+      Promise.all(
+        endpointTypeDeviceTypeRefPair.deviceTypeRef.map((ref) =>
+          axiosRequests.$serverGet(`${restApi.uri.deviceTypeClusters}${ref}`)
         )
-        .then((res) => {
-          setRecommendedClusterList(context, res.data)
-        })
+      ).then((responses) => {
+        const allClusters = responses.flatMap((res) => res.data)
+        setRecommendedClusterList(context, allClusters)
+      })
     )
     p.push(
-      axiosRequests
-        .$serverGet(
-          `${restApi.uri.deviceTypeAttributes}${endpointTypeDeviceTypeRefPair.deviceTypeRef}`
```

Comment:
Why did you do this? I'm lost

Reply:
This was all broken when multiple device type refs are passed(Which happens when user selects more than one device type on an endpoints). The URIs were like `Identify/1,2` which would return nothing. Now it does `Identify/1 and `Identify/2` and collates the 2 results.

Am surprised no one has complained about this yet.